src/test/java/io/openliberty/tools/intellij/it/SingleModMPProjectTestCommon.java

```diff
@@ -1007,6 +1029,59 @@ public static void deleteDirectoryIfExists(String dirPath) {
         }
     }
 
+    /**
+     * Clean project.
+     */
+    public void stopTerminal() {
+
+        Keyboard keyboard = new Keyboard(remoteRobot);
+        ProjectFrameFixture projectFrame = remoteRobot.find(ProjectFrameFixture.class, Duration.ofSeconds(10));
+        ComponentFixture terminal = remoteRobot.find(ComponentFixture.class, byXpath("//div[@class='JBTerminalPanel']"), Duration.ofSeconds(10));
+
+        terminal.rightClick();
+        ComponentFixture openFixtureNewTab = projectFrame.getActionMenuItem("New Tab");
+        openFixtureNewTab.click(new Point());
+
+        // Perform Stop Action
+        if (getBuildCategory() == BuildType.MAVEN_TYPE) {
+            keyboard.enterText("./mvnw liberty:stop");
+
+        } else if (getBuildCategory() == BuildType.GRADLE_TYPE) {
+            keyboard.enterText("./gradlew libertyStop");
+        }
+        keyboard.enter();
+        TestUtils.sleepAndIgnoreException(10);
+    }
+
+    /**
+     * Stop the Server.
+     */
+    public void cleanTerminal() {
+
+        Keyboard keyboard = new Keyboard(remoteRobot);
+        // Perform clean
+        if (getBuildCategory() == BuildType.MAVEN_TYPE) {
+            keyboard.enterText("./mvnw clean");
+            keyboard.enter();
```

Comment:
I'm not sure you need Enter here if you type Enter right after the IF.

Reply:
The enterText method inputs the text, but without the enter() command, it won't trigger the command to run.

Reply:
Yes, but there is also Enter on line 1074. This is similar to line 1054.

Reply:
Ok. I have modified that